component/azstorage/datalake_test.go

```diff
@@ -1164,40 +1168,39 @@ func (s *datalakeTestSuite) TestRenameFile() {
 	s.assert.Nil(err)
 
 	// Src should not be in the account
-	source := s.containerUrl.NewDirectoryURL(src)
-	_, err = source.GetProperties(ctx)
+	source := s.containerClient.NewDirectoryClient(src)
+	_, err = source.GetProperties(ctx, nil)
 	s.assert.NotNil(err)
 	// Dst should be in the account
-	destination := s.containerUrl.NewDirectoryURL(dst)
-	_, err = destination.GetProperties(ctx)
+	destination := s.containerClient.NewDirectoryClient(dst)
+	_, err = destination.GetProperties(ctx, nil)
 	s.assert.Nil(err)
 }
 
 func (s *datalakeTestSuite) TestRenameFileMetadataConservation() {
 	defer s.cleanupTest()
 	// Setup
 	src := generateFileName()
-	source := s.containerUrl.NewRootDirectoryURL().NewFileURL(src)
+	source := s.containerClient.NewFileClient(src)
 	s.az.CreateFile(internal.CreateFileOptions{Name: src})
 	// Add srcMeta to source
-	srcMeta := make(map[string]string)
-	srcMeta["foo"] = "bar"
-	source.CreateWithOptions(ctx, azbfs.CreateFileOptions{Metadata: srcMeta}, azbfs.BlobFSAccessControl{})
+	srcMeta := make(map[string]*string)
+	srcMeta["foo"] = to.Ptr("bar")
+	source.SetMetadata(ctx, srcMeta, nil)
```

Comment:
Should this call be Create or SetMetadata?

Reply:
It should be the SetMetaData function only I think as the Create function creates the file taking createoptions as input not setting metadata